lib/meadow/data/schemas/field.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Meadow.Data.Schemas.Field do
+  @moduledoc """
+  Field schema.
+  """
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  import EctoEnum
+
+  defenum(MetadataClassEnum,
+    administrative: "administrative",
+    core: "core",
+    descriptive: "descriptive"
+  )
+
+  {@primary_key({:id, :string, autogenerate: false})}
```

Comment:
Why is this whole thing wrapped in `{}`? Do we do that elsewhere? Is it a best practice?

Reply:
Must have been a copy pasta thing, fixed!